src/test/rpc/JSONRPC_test.cpp

```diff
@@ -2160,6 +2160,32 @@ class JSONRPC_test : public beast::unit_test::suite
             BEAST_EXPECT(RPC::contains_error(result));
             BEAST_EXPECT(!req[jss::tx_json].isMember(jss::Fee));
         }
+
+        {
+            // transaction with a higher base fee
+            Json::Value req;
+            test::jtx::Account const alice("alice");
+            Json::Reader().parse(
+                "{ \"tx_json\" : { \"TransactionType\": \"AccountDelete\", "
+                "\"Account\": \"" +
+                    env.master.human() + "\", \"Destination\": \"" +
+                    alice.human() + "\" } } ",
+                req);
+            Json::Value result = checkFee(
+                req,
+                Role::ADMIN,
+                true,
+                env.app().config(),
+                feeTrack,
+                env.app().getTxQ(),
+                env.app());
+
+            BEAST_EXPECT(!RPC::contains_error(result));
```

Comment:
Check that `result` is empty object (which it should be for success)

Reply:
Fixed in 1c0b3a6f8ca9b038035689b68971ad5e31ba30df